README.md

```diff
@@ -2220,6 +2220,15 @@ form-flow:
 | `form-flow.uploads.thumbnail-width`     | `64`                                                       | Thumbnail width in pixels                                                                                                             |
 | `form-flow.uploads.thumbnail-height`    | `60`                                                       | Thumbnail height in pixels                                                                                                            |
 
+##### Virus scanner properties
+
+| Property                                                | Default | Description                                                                                                                                        |
+|---------------------------------------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `form-flow.uploads.virus-scanning.enabled`              | `true`  | Turns on virus scanning component                                                                                                                  |
+| `form-flow.uploads.virus-scanning.clammit-url`          | None    | Full path for scanning on hosted (clamav-server)                                                                                                   |
```

Comment:
I'm not sure we want to call this `clammit-url` - maybe just `service-url`. Do we want this to be generic to be used by any implementation of a scanning service?

Reply:
(I realize this is a sw arch question and not just a doc thing)

Reply:
Good idea, I'll switch to `service-url`